.github/workflows/automerge.yml

```diff
@@ -0,0 +1,22 @@
+name: Automerge
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+
+jobs:
+  automerge:
+    name: Automerge
+    runs-on: ubuntu-latest
+    steps:
+      - id: automerge
+        name: automerge
+        uses: "pascalgn/automerge-action@v0.16.2"
+        env:
+          # This needs to be a PAT to trigger the on-commit pipeline
+          GITHUB_TOKEN: "${{ secrets.ACCESS_TOKEN }}"
+          MERGE_DELETE_BRANCH: "true"
+          MERGE_FILTER_AUTHOR: "github-actions[bot]"
```

Comment:
I doubt this is very secure

Reply:
This was made under the assumption that a branch rule requires at least 1 approval of code owners.